tools/tests/form-control/test.js

```diff
@@ -304,7 +270,64 @@ const forms = [
 				</gameface-form-control>
 				<h3>Response data:</h3>
 				<span class="response" id="${RESPONSE_CONTAINER_ID}"></span>
-			</div>`
+			</div>`,
+};
+
+const forms = [
+	{
+		testName: formsTestNames.LOGIN_FORM,
+		id: formsIds.LOGIN_FORM,
+		shouldAddRequestFinishCallback: true,
```

Comment:
TL;DR
Removed along with the usage (it was used).

The purpose was to avoid adding and event listener on the form that doesn't need it because it is not submitted but it doesn't interfere with anything if it is added.